karma.conf.babel.js

```diff
@@ -0,0 +1,80 @@
+import paths from './paths';
```

Comment:
Just noticed we have [`babel-core`](https://github.com/TechnologyAdvice/stardust/pull/132/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R43) as a dependency so we should still be able to `babel-node` it from `package.json` if desired.